### PR TITLE
Retrieve the errors properly from the back end

### PR DESCRIPTION
**Pull request by charliearlie**

### CODE REVIEW

File: src/components/Authentication/RegistrationStepper.tsx [hunk 1]
```diff
@@ -174,33 +176,45 @@ class RegistrationStepper extends React.Component<any, IState> {
...
+          email: this.state.basicDetails.email,
+          password: this.state.basicDetails.password,
+          confirmPassword: this.state.basicDetails.confirmPassword,
+        }),
+      });
+
+      const { errors, user } = res.data;
+
+      if (errors) {
+        console.log(errors);
```
Comment (LorenzoKoundouris) [inline src/components/Authentication/RegistrationStepper.tsx]: intentional log?

File: src/components/Authentication/RegistrationStepper.tsx [hunk 2]
```diff
@@ -174,33 +176,45 @@ class RegistrationStepper extends React.Component<any, IState> {
...
       setTimeout(() => {
-        this.props.cookies.set(LOGGED_IN_USER, res.data.email);
-        this.props.cookies.set(TOKEN, res.data.token);
+        this.props.cookies.set(LOGGED_IN_USER, user.email);
+        this.props.cookies.set(TOKEN, user.token);
         this.props.closeDialog();
       }, 500);
-    });
+    } catch (error) {
+      console.error(error);
```
Comment (LorenzoKoundouris) [inline src/components/Authentication/RegistrationStepper.tsx]: A good read..
https://medium.com/@dhruvrajvanshi/making-exceptions-type-safe-in-typescript-c4d200ee78e9

File: src/components/Authentication/RegistrationStepper.tsx (hunk 1 above)
Comment (charliearlie) [reply src/components/Authentication/RegistrationStepper.tsx]: Removed it

File: src/components/Authentication/RegistrationStepper.tsx (hunk 2 above)
Comment (charliearlie) [reply src/components/Authentication/RegistrationStepper.tsx]: It's more just for debugging for now. It will be removed.

I'll read the article too 👀

**Pull request merged**